kong/db/dao/plugins/mp_rpc.lua

```diff
@@ -0,0 +1,123 @@
+local msgpack = require "MessagePack"
```

Comment:
Move this file out of `kong/db` as well, I suggest inside `kong/runloop/plugin_servers`